OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - Fix Badges' Display Issue

# Description
Fix the issue of badges that cannot be displayed on the weekly summaries report page after it is assigned.

## Main changes explained:
- Ignore the property `showReport` to display badges
- Fixed lint errors

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/owner user
4. Assign one or more badges to a user
5. go to dashboard→ Report → Weekly Summaries Report
6. Check whether the badge is shown and displayed in the **correct tab**.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/c04429f8-0c65-457f-aca0-14d24e0ec5f8
